webassembly/app.html

```diff
@@ -223,6 +221,49 @@ <h1 class="title">F3D Web</h1>
         // do a first render and start the interactor
         Module.engineInstance.getWindow().render();
         Module.engineInstance.getInteractor().start();
+
+        // Parse search-query mesh url-param or load default mesh file
+        // const params = new URLSearchParams(window.location.search);
+        // Replace first hash with question mark to have real search query parsing and avoid leading # in first parsed urlparam
+        const params = new URLSearchParams(window.location.hash.replace('#', '?'));
```

Comment:
```suggestion
        const params = new URLSearchParams(window.location.hash.replace(/^#/, ''));
```
to remove only the `#` at the beginning